src/main/java/frc/robot/subsystems/Camera.kt

```diff
@@ -73,9 +73,10 @@ class Camera(val name: String, val pose: Transform3d) {
             if (validTargets.size > 1) currentStds = multiTagStds
 
             currentStds =
-                if (validTargets.size == 1 && avgDist > 4)
+                if (/*validTargets.size == 1 && avgDist > 7*/ false)
                     VecBuilder.fill(Double.MAX_VALUE, Double.MAX_VALUE, Double.MAX_VALUE)
                 else currentStds.times(1 + (avgDist.pow(2) / 30))
+            //            currentStds.times(1 + (avgDist.pow(2) / 30))
```

Comment:
Can you remove the commented out code, and maybe add a comment that we should look into excluding further away targets?